db/migrate/20171116140837_initialize_collaboration_amounts.rb

```diff
@@ -1,6 +1,10 @@
 class InitializeCollaborationAmounts < ActiveRecord::Migration[5.1]
+  class Collaboration < ApplicationRecord
+    self.table_name = :decidim_collaborations_collaborations
+  end
+
```

Comment:
Interesting! I faced scenarios like this one. This is to avoid problems with data migrations and the data model when the table is changing?

Reply:
If you load the real model during migrations, and old version of the model will be loaded (without the reamaining migrations applied). I was getting a crash during seeding because of this where a model column would not be found.

Reply:
👍